### DISCUSSION TITLE
Welcome to cugyen Discussions!

### DISCUSSION THREAD

**Discussion by ugyenchoden** [cugyen/resume#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ugyenchoden** [comment]
any suggestions?

**Reply by prabarai** [reply]
add graph please

**Comment by prabarai** [comment]
did u code in ruby?

**Reply by ugyenchoden** [reply]
nope it is not.

**Comment by prabarai** [comment]
please speak up

**Reply by ugyenchoden** [reply]
sorry what?

**Comment by prabarai** [comment]
i no english
